jdaviz/core/loaders/importers/spectrum_list/spectrum_list.vue

```diff
@@ -1,5 +1,37 @@
 <template>
   <v-container>
+    <div v-if="!disable_dropdown && exposures_items.length > 0">
+      <plugin-select
+        :items="exposures_items.map(i => i.label)"
+        :selected.sync="exposures_selected"
+        :show_if_single_entry="true"
+        :multiselect="exposures_multiselect"
+        :search="true"
+        label="Exposure"
+        api_hint="ldr.exposures.selected ="
+        :api_hints_enabled="api_hints_enabled"
+        hint="Exposures to select from."
+      ></plugin-select>
+    </div>
+
+    <div style="height: 8px;"></div>
+
+    <div v-if="!disable_dropdown">
+      <plugin-select
+        :items="spectra_items.map(i => i.label)"
+        :selected.sync="spectra_selected"
+        :show_if_single_entry="true"
+        :multiselect="spectra_multiselect"
+        :search="true"
+        label="Source IDs"
```

Comment:
maybe if this is the intuitive label, that the attribute should be called `sources` instead of `spectra`?